docs/user/feeds.md

```diff
@@ -742,7 +742,7 @@ parameters:
   port: 61614
   provider: CERT.PL
   server: n6stream-new.cert.pl
-  ssl_ca_certificate:
+  ssl_ca_certificate: 
```

Comment:
```suggestion
  ssl_ca_certificate:
```
doesn't look intentional

Reply:
Fixed. But it was (re)generated this way with `scripts/generate-feeds-docs.py`.